### PR TITLE
Re-add missing SIGSTKSZ defines from musl 1.1.15

### PR DESCRIPTION
**Pull request by jgravelle-google**
Fixes https://github.com/kripken/emscripten/issues/5146

### CODE REVIEW

File: system/lib/libc/musl/arch/emscripten/bits/signal.h [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #if defined(_POSIX_SOURCE) || defined(_POSIX_C_SOURCE) \
  || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) || defined(_BSD_SOURCE)
 
+#if defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) || defined(_BSD_SOURCE)
```
Comment (kripken) [inline system/lib/libc/musl/arch/emscripten/bits/signal.h]: hmm, are any of those defined for us?

Comment (jgravelle-google) [inline system/lib/libc/musl/arch/emscripten/bits/signal.h]: Clang apparently defines `_XOPEN_SOURCE` by default: https://github.com/llvm-mirror/clang/blob/master/lib/Basic/Targets.cpp#L697

Comment (kripken) [inline system/lib/libc/musl/arch/emscripten/bits/signal.h]: Oh, ok. Cool, lgtm.

**Comment by jgravelle-google** [comment]
Should we bump the version number for this so users get a rebuilt libc by default?

**Comment by kripken** [comment]
This just affects a header, and the error showed up as a compile time error, so I think it's ok without. Unless this change lets libc get built differently? We can ask the user to confirm it works, and see about bumping the version number later.

**Comment by jgravelle-google** [comment]
Oh, right, cool. Thanks!

**Pull request merged**